detector_test.go

```diff
@@ -622,22 +997,21 @@ func testDetector(t *testing.T, when spec.G, it spec.S) {
 		})
 
 		it("should fail if the group is empty", func() {
-			_, _, err := resolver.Resolve([]buildpack.GroupBuildpack{}, &sync.Map{})
+			_, _, err := resolver.Resolve([]buildpack.GroupElement{}, &sync.Map{})
 			if err != lifecycle.ErrFailedDetection {
 				t.Fatalf("Unexpected error:\n%s\n", err)
 			}
 
 			if s := cmp.Diff(h.AllLogs(logHandler),
 				"======== Results ========\n"+
-					"Resolving plan... (try #1)\n"+
```

Comment:
We don't need to try to resolve the plan to know that there are no buildpacks in the group.